src/en/news/blog/2022/ceph-use-case-survey-2022/index.md

```diff
@@ -0,0 +1,123 @@
+
+<center>Fig. 3: A pie-chart breakdown of survey participants' willingness to share their cluster use case publicly</center><br><br>
+    
+## Use Cases
+
+Of the sixty-eight total responses, fifty-eight people volunteered to share the use case of their cluster. We were very pleased with the variety of responses we received.
+
+![A word cloud highlighing most frequent key words from the user responses. Top 6 key words are "storage", "ceph", "cephfs", "rbd", "openstack", and "data".](images/word_cloud_2.png)
+
+<div align="center">The figure above is a word cloud highlighing most frequent key words from the user responses.</div><br>
+
+### Several people reported using Ceph for research purposes:
+
+- One anonymous Ceph user reported that CephFS is currently used as the backend of the [IRI Data Library](https://iri.columbia.edu/resources/data-library/), which is used to store about 700TB of climate, geophysical, health and agriculture data in multiple formats. They chose Ceph to handle the millions of data downloads their servers experience every month.
+- Ceph user Andrew Ferris wrote that Ceph is used as a large-scale storage solution at his research facility. As said by Andrew, "Ceph's ability to run on cheaper commodity hardware and scalability is valuable to us."
+- Lance Joseph, another Ceph user, explained that Ceph is used to serve large scale (+10 petabytes) turbulence and oceanography data sets for researchers and scientists. This data is backed by CephFS and S3 object storage.
+- Finally, Ceph user Mike Lowe mentioned using Ceph as backing storage for National Science Foundation researchers.
+
+### Others reported using Ceph in testing and development environments:
+
+- One anonymous Ceph user described using the S3 API to conduct local testing of applications and microservices.
+- Kamil Madac, a Ceph user, wrote that Ceph acts as OpenStack private cloud storage for a company with 100 employees. The private cloud is used for internal production VMs and for dev/test environments for apps that are developed.
+- Kamoltat Sirivadhna, a Ceph Developer, reported using Ceph for development purposes.
+- Goutham Pacha Ravi, an integration enineer who develops/maintains driver interfaces to CephFS in OpenStack and Kubernetes, described using Ceph for early integration testing.
+- Finally, open source developer Ngwa Secrick Meh wrote about using Ceph clusters for development in the Ceph Dashboard.
+
+### Oftentimes, Ceph users employ multiple interfaces at once.
+
+Ceph user Enrico Bocchi of CERN described one such scenario:
+
+> At CERN, we use Ceph for block, object, and filesystem storage. The main use cases are:
+>
+> - RBDs for OpenStack VMs (bootable volumes, storage volumes, glance images)
+> - RBDs for other Storage services (e.g., AFS and NFS are provided by running VMs with RBDs attached used as backend storage for the overlaying service)
```

Comment:
This is a direct quote from Enrico. I get how some parts of it can be confusing though. I'll try paraphrasing it and removing the quote block so it's clear that we've paraphrased instead of direct-quoted.

Reply:
Cool. Don't wait on me to get back to you with a rubber stamp of approval. I have complete confidence in your judgment, and you already have the bureaucratic GitHub PR approval that you need from me to merge this PR, so don't feel that I have to be consulted on this matter. When it looks right to you, it's right.